app/components/tooltip-renderer.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@ember/component';
+import { computed } from '@ember-decorators/object';
+import { argument } from '@ember-decorators/argument';
+import mustache from 'npm:mustache';
+
+export default class TooltipRenderer extends Component {
+  @computed('feature', 'template')
```

Comment:
Only danger here is mutability - this computed will only break cache and trigger re-render when the whole feature is replaced, but not the properties (which are references in the getter). Since the feature being passed is always replaced anyways, I don't think it'll matter

Reply:
Good to know